extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -71,7 +59,8 @@
   padding: 0 !important;
   border: 0 !important;
   background-color: transparent !important;
-  transition: transform 0.225s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
+  filter: drop-shadow(0px 0px 16px rgba(0, 0, 0, 0.5)) !important;
```

Comment:
Re: 12px

Also, why aren't we using the same values as for the tooltip?

Reply:
These were @hongwei1990's instructions

Reply:
What is the 12 we r talking about here?